### PR TITLE
Add HTML and CSS

### PR DESCRIPTION
**Pull request by richy4dev**
In this repo, I reviewed my project from microverse trials.
I setup linters correctly
I checked for linters error.
I reviewed the project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Registeration Form</title>
+    
+    <link rel="stylesheet" href="Styles.css">
```
Comment (richy4dev) [inline index.html:9]: In HTML, kindly rename this to match your stylesheet "style.css" so that your css file would be linked to your html file

**Review by isaacpitwa — CHANGES_REQUESTED** [review]
**Hi @richy4dev 👋🏿** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there 🏅 



## Highlights ✔️ 
- Comments are clear and easy to understand ✔️
- Proper Use of Inline Comments ✔️
- Well explained reason why suggested Changes ✔
## Required changes ♻️ 

- [ ]  it is required that you write at least two reviews. which in your case you have only one 🫢. please add at least one more review. Refer to the  [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/review_your_project.md).

**Kindly add all other comments as inline comments this helps the summary be easy to read and they can always go back to the inline comments to fix required changes**

You can also add how the student can reach you on slack or by tagging you in their questions

File: index.html [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+  </div>
+
+  <!--
+  <label class="tacbox">
+    <input type="checkbox" />
+    I agree to these <a href="#">Terms and Conditions</a>.
+  </label>
+  
+  <label for="taccheck" class="tacbox">
+    <input id="taccheck" type="checkbox" />
```
Comment (richy4dev) [inline index.html:146]: In your css file, Kindly set the display of your "tacbox" class so the checkbox can display on the same row as the terms and conditions. Preferably use flex.

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @richy4dev ✋



Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Optional suggestions

- [ ] Kindly use emojis as they are more human-friendly.
- [ ] You may also use more polite communication. Always make suggestions instead of commands or orders

##### _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

##### Cheers and Happy coding!:clap::clap::clap:
##### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**